Update CONTRIBUTING with copy from release notes

rogeruiz opened the pull request:
I ported over the `gulp workflow` section from the release notes into what was already in CONTRIBUTING. I updated the release notes [here](https://github.com/18F/web-design-standards/releases) with the a reference to this update.

rogeruiz commented:
@kategarklavs I only added to what was already here in the contributing docs, but if you could take a look at this that'd be good.

kategarklavs commented:
@rogeruiz Taking a look now!

kategarklavs commented on CONTRIBUTING.md:
```diff
@@ -46,50 +46,60 @@ Here are a few guidelines to follow when submitting a pull request:
 
 Questions or need help with setup? Feel free to open an issue here [https://github.com/18F/web-design-standards/issues](https://github.com/18F/web-design-standards/issues).
```
I'd replace "Questions" with "Have questions or ..." 

Also, to promote concision, I'd remove the "Feel free to" and start the second sentence with "Open an issue here:"

kategarklavs commented on CONTRIBUTING.md:
```diff
@@ -46,50 +46,60 @@ Here are a few guidelines to follow when submitting a pull request:
 
-Once you have Gulp installed globally, you can run the following command to
-see all the commands available to you.
+To start, you’ll need to run the following command to
```
To tighten things up a bit, I might recommend starting this sentence as follows:

"To start, run the following command to ..."

kategarklavs commented on CONTRIBUTING.md:
```diff
@@ -46,50 +46,60 @@ Here are a few guidelines to follow when submitting a pull request:
-supported via `npm`.  This package contains no documentation and the compiled
-and bundled CSS, JavaScript, fonts, and images. To build the package locally,
-run the following command:
+To help you get started, here are a few of the common tasks that you may find
```
Would it be OK to remove "To help you get started" here, and instead begin the sentence with "Here are a few of the common tasks ..."

kategarklavs commented on CONTRIBUTING.md:
```diff
@@ -46,50 +46,60 @@ Here are a few guidelines to follow when submitting a pull request:
-with the following command. This will setup Gulp and Jekyll to watch for file
-changes to the `docs/` and `src/` directory and update the website accordingly.
+The task above builds the entire Draft U.S. Web Design Standards website locally.
+It can be useful when debugging for build errors. Building the Draft U.S. Web
```
Could we change this to active voice (best practice)? 

For example:

"You can build the Draft U.S. Web Design Standards website using gulp with the following command:"

kategarklavs commented:
@rogeruiz Hey! Just took a look and made a few comments/recommendations. If you have any questions, please send them my way!

kategarklavs commented on CONTRIBUTING.md:
```diff
@@ -46,50 +46,58 @@ Here are a few guidelines to follow when submitting a pull request:
-supported via `npm`.  This package contains no documentation and the compiled
-and bundled CSS, JavaScript, fonts, and images. To build the package locally,
-run the following command:
+Here are a few of the common tasks:
```
Ah, OK: So, as I was reading this, I could see how a user might get confused by the statement "Here are a few of the common tasks," which is followed by one example (gulp build).

What if we were to reword this to "The following examples cover a few common tasks" -OR- "The following examples detail a few tasks you'll encounter as you use gulp" — something along those lines?

msecret commented on CONTRIBUTING.md:
```diff
@@ -46,50 +46,58 @@ Here are a few guidelines to follow when submitting a pull request:
-Working on the Draft Web Design Standards site locally can be done via `gulp`
-with the following command. This will setup Gulp and Jekyll to watch for file
-changes to the `docs/` and `src/` directory and update the website accordingly.
+The task above builds the entire Draft U.S. Web Design Standards website locally.
```
Should we now call this "Standards website"

rogeruiz replied:
@msecret yup! @juliaelman and I are working on a set of PRs that address the `UI Component` and `Standards website` in the `/docs/**/*` and `*.md` files

rogeruiz commented:
@kategarklavs and @msecret I've updated the branch and addressed your comments :tada: let me know if there is anything else this needs before we can merge it in. Thanks! :rocket:

kategarklavs commented:
@rogeruiz Checking now! Thanks for the heads up.

kategarklavs commented on CONTRIBUTING.md:
```diff
@@ -44,52 +44,61 @@ Here are a few guidelines to follow when submitting a pull request:
-supported via `npm`.  This package contains no documentation and the compiled
-and bundled CSS, JavaScript, fonts, and images. To build the package locally,
-run the following command:
+The following examples detail a few tasks you'll encounter as you use gulp
```
Please add a colon after "as you use gulp," so the sentence will read as follows:

The following examples detail a few tasks you'll encounter as you use gulp:

kategarklavs commented:
@rogeruiz OK, just took a look and made one comment about adding a colon — aside from that, this looks good to go :+1:

msecret commented on package.json:
```diff
@@ -4,12 +4,13 @@
   "main": "src/js/start.js",
   "scripts": {
-    "prepublish": "gulp build",
+    "prepublish": "bundle && gulp build",
```
I personally think that we shouldn't require ruby as a dependency to work on the project and we should just put `gulp no-test build` here, but am fine with this for now.

rogeruiz replied:
yeah I agree with this, but the `/docs` directory is still a Ruby project so the `Gemfile` being there is what makes it a ruby project. 0.2% is still something I guess.

<img width="985" alt="screen shot 2016-03-11 at 2 07 13 pm" src="https://cloud.githubusercontent.com/assets/706004/13712608/9f57fe30-e792-11e5-9585-5a566688438b.png">

msecret commented:
LGTM
